src/biome/text/_model.py

```diff
@@ -277,6 +275,38 @@ def predict(self, *args, **kwargs) -> Dict[str, numpy.ndarray]:
 
         return prediction
 
+    def predict_batch(
+        self, input_dicts: Iterable[Dict[str, Any]],
+    ) -> List[Dict[str, numpy.ndarray]]:
+        """Returns predictions given some input data based on the current state of the model
+
+        The predictions will be computed batch-wise, which is faster
+        than calling `self.predict` for every single input data.
+
+        Parameters
+        ----------
+        input_dicts
+            The input data. The keys of the dicts must comply with the `self.inputs` attribute
+
+        Returns
+        -------
+        predictions
+        """
+        if self.training:
+            warnings.warn(
+                "Train model enabled. "
+                "Disabling training mode automatically. You can manually disable it: "
+                "self.eval() or self.train(False)"
+            )
+            self.eval()
+
+        instances = [self.text_to_instance(**input_dict) for input_dict in input_dicts]
```

Comment:
Just a comment but we must handle this at some point:

Since method `text_to_instance` could return an empty instance, we could have problems applying batch prediction here.

We had this problem in older versions. 

Maybe a metadata field keeping instance index in batch could resolve this problem but we must discuss about